manifests/b/brimdata/brim/0.31.0/brimdata.brim.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: brimdata.brim
+PackageVersion: 0.31.0
+PackageLocale: en-US
+Publisher: Brim Security, Inc.
+PublisherUrl: https://github.com/brimdata/brim
+PublisherSupportUrl: https://github.com/brimdata/brim/issues
+# PrivacyUrl: 
+Author: Brim Security, Inc.
+PackageName: Brim
+PackageUrl: https://www.brimsecurity.com
+License: Copyright © 2021 Brim Security, Inc.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or other terms as appropriate